tasks/validate.yml

```diff
@@ -7,8 +7,3 @@
   run_once: true
   delegate_to: localhost
   become: false
-
-- name: 'Validate OS Family'
```

Comment:
Let's add to README that this role available only on "RedHat", "Debian" families